docs/zh-Hans/Tutorials/Todo/Index.md

```diff
@@ -0,0 +1,830 @@
+# 快速入门
+
+````json
+//[doc-params]
+{
+    "UI": ["MVC", "Blazor", "BlazorServer", "NG"],
+    "DB": ["EF", "Mongo"]
+}
+````
+
+这是一个由单个部分组成的快速入门教程，旨在使用ABP框架构建一个简单的待办事项应用程序。 这是最终应用程序的屏幕截图：
+
+![待办事项列表](todo-list.png)
+
+你可以在[这里](https://github.com/abpframework/abp-samples/tree/master/TodoApp)找到已完成的项目源代码。
+
+## 先决条件
+
+* 一个集成开发环境 (比如: [Visual Studio](https://visualstudio.microsoft.com/vs/)) 它需要支持 [.NET 5.0+](https://dotnet.microsoft.com/download/dotnet) 的开发。
```

Comment:
。 => .

We use half-width characters

Reply:
OK, are there any other similar requirements? thank you

Reply:
Always use half-width characters